helix-term/src/keymap/default.rs

```diff
@@ -363,7 +363,7 @@ pub fn default() -> HashMap<Mode, Keymap> {
         "A-d" | "A-del" => delete_word_forward,
         "C-u" => kill_to_line_start,
         "C-k" => kill_to_line_end,
-        "C-h" | "backspace" => delete_char_backward,
+        "C-h" | "backspace" | "S-backspace" => delete_char_backward,
```

Comment:
are there other keybindings that might have worked without this feature but don't work anymore now?
If that is the case should we offer a config option to dsiable this feature? Also I think this technically qualifies as a breaking change because people could have custom keybdings like this which would stop working. I added the apropriate label so we don't forget about that

Reply:
Ambiguous keycodes like using `C-i` for `tab` or using `C-backspace` for `C-h` would change but I think those would be pretty rare. I'm not sure how I feel about a config option for this: if the protocol is enabled then you get more predictable and correct key events - you would only disable it if you wanted to use ambiguous keycodes which I don't think has any advantage

Reply:
sounds reasonable :+1: I think we should keep the breaking change label tough so this PR is a bit easier to find for people who do experience regressions.